### PR TITLE
Add settings dialog using addon-menubar-items Fixes #8

### LINKED ISSUE #8: Make sharemp dynamic parameter (read by property) + GUI
> To enable it go to landing_rate folder and open addon-config.xml. Find sharemp line.
> <sharemp>0</sharemp>
> Change sharemp value (1 for yes and 0 for no). Restart your simulator. Done.

It would be nice if:

- [x] `<sharemp>` would be dynamically read from a property below `/addons/by-id/landing_rate`
- [x] default to `false`
- [x] setting to `true` therefore would, from that point onwards, share it in the chat. Setting to `false` again would disable sharing from that point onwards.
- [x] a small config GUI would provide access to that property for normal users.
- [x] correct readme.md 

```nasal
# Init GUI menu entry
    var menuTgt = "/sim/menubar/default/menu[7]";  # 7=multiplayer
    var menudata = {
		label   : "Landing Rate",
		name    : "landing_rate",
		binding : { command : "dialog-show", "dialog-name" : "landing_rate_settings" }
	};
	props.globals.getNode(menuTgt).addChild("item").setValues(menudata);
	fgcommand("gui-redraw");
```
Re: the menu bar index; I believe there is a standardized "Addons" menu index we should use.

### PR DESCRIPTION
**Pull request by RenanMsV**

### CODE REVIEW

**Comment by hbeni** [comment]
Looks good!

Is the setting persistent across sessions already?
If not, what is your opinion about that?

**Comment by RenanMsV** [comment]
No its not persistent. Making it persistent would be better yes.

Edit
Just made it persistent with the new commit

**Comment by hbeni** [comment]
> Making it persistent would be better yes.

Well, there are also arguments against it... Because if i forgot to disable it before closing FGFS, i may spill out all landings to others.
So maybe, the best way is to make this depending on a (persistent) user choice? :)

(A qick&easy solution could be make both boxes persistent, and when the "save the setting" box is _not_ ticket, unset the "show in MP" flag)

**Comment by hbeni** [comment]
OTOH, I could also just disable the box at the first place when thinking about "if i want this stored".
So probably its the best thing you just keep it.

Or: Display a status message when the addon is loaded, so we get feedback at the startup.
Thinking about it more, this seems to be the best solution

**Comment by RenanMsV** [comment]
So I will add a message right after the addon loading reminding the user sharemp is enabled.

**Comment by RenanMsV** [comment]
I just added the warning. Looks like its ok to merge.

**Comment by hbeni** [comment]
Great, looks very good!
Yellow is a decent choice.

File: README.md [hunk 1]
```diff
@@ -30,14 +30,7 @@ If you're using an incompatible aircraft a message will be show. And the addon w
...
 
-If you want, you can activate it. And a message will be sent by multiplayer chat showing your landing stats.
-
-To enable it go to landing_rate folder and open addon-config.xml. Find sharemp line.
-
-```<sharemp>0</sharemp>```
-
-Change sharemp value (1 for yes and 0 for no). Restart your simulator. Done.
-
+The addon can automatically send your landing rate in a message if you are in a multiplayer server. You can activate it in the addon settings menu, that can be found by clicking the Landing Rate option of your menu bar.
```
Comment (hbeni) [inline README.md]: Maybe add a sentence that explains that it's persited across sessions?

Comment (RenanMsV) [reply README.md]: Yes you are right.

Comment (RenanMsV) [reply README.md]: Added

**Pull request merged**